pan-domain-auth-play_2-4-0/src/main/scala/com/gu/pandomainauth/action/Actions.scala

```diff
@@ -117,10 +117,18 @@ trait AuthActions extends PanDomainAuth {
 
     // use legacy cookie in the check for now, even though both are dropped
     // phase two of the rollout will be to use the assymetric cookie
+    /**
```

Comment:
You should replace the comment above with something along the lines of "it's now time to do both checks".